server/.env

```diff
@@ -1,3 +1,5 @@
 TEAM_NAME="Aecio, Shums, Marco, Test, Ramazan, Yasuko, Kartihan, Shawn"
 JWT_SECRET="change-for-production"
 
+AWS_SECRET_KEY=“7GJlnP9nM6MhuInFFBmJbT/7+TO1IxswnZlcI2/u”
+AWS_ACCESS_KEY=“AKIAJ6MUQQHJBDA63PFQ”
```

Comment:
Pls delete this asap. Secret keys should be kept locally. This will be added during production deployment